### DISCUSSION TITLE
Have a `tool.poetry.group.torch` for dependencies -- I'd like to source 'em all from Pytorch.org (surprise, surprise!) and configure that @ the group level, is this possible?

### DISCUSSION THREAD

**Discussion by BuildBackBuehler** [python-poetry/poetry#9340] [Q&A]
I've read through all the docs/searched the issues/requests/quick interwebs dive. I suppose I should feel spoiled as-is after a long, painful stint with Pipenv. I have no clue why I went back to it. Y'anyways, I was a bit disappointed because I tried making some declarations:

[tool.poetry.group.torch]
optional = False
source = "Torch" (how I named pytorch.org in [[t.p.source]])

Followed by [t.p.g.t.dependencies] and names of packages, i.e., Colorama = "*". 

Got an error and provided no alternative as to how I could custom configure at a group level. 

I'd really like to cutdown on fluff/errors by configuring/curating groups of dependencies with a catch-all source or however else makes sense in the future. Maybe the ability to change the source priority from its global "supplemental" to "explicit/default". A non-source one...and this may be possible currently would be to create package markers that are applied to each package in the group. 

Of course, with the CLI you could install all these Pytorch packages into your group with the --source "pytorch.org" argument as a workaround but I'd still be left with a per-package customization and I find those tend to break...in Pipenv heh. 

Thanks for any help!

**Comment by dimbleby** [comment]
#8138
